streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -255,6 +255,7 @@ private static StreamsConfig createConfig(final String eosConfig, final String e
         return createConfig(eosConfig, enforcedProcessingValue, deserializationExceptionHandler, LogAndFailProcessingExceptionHandler.class.getName());
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Don't we need to update the code, too, to use `DESERIALIZATION_EXCEPTION_HANDLER_CLASS_CONFIG` instead of `DEFAULT_DESERIALIZATION_EXCEPTION_HANDLER_CLASS_CONFIG`?

Reply:
But after this change, I notice 3 other tests are failing. Strange, the config is not set properly, and originals are 0 in TopologyConfig. Not sure what am I missing. Can you pls take a look.

Reply:
Just needed a little bit of debugging... -> https://github.com/apache/kafka/pull/17005/files#r1737574338